viewing_party/main.py

```diff
@@ -0,0 +1,165 @@
+        genre_list.append(key['genre'])
+    genre_dict = {}
+
+    for genre in genre_list:
+        if genre in genre_dict:
+            genre_dict[genre] = genre_dict[genre] + 1 
+        else:
+            genre_dict[genre] = 1
+    genre_count_list = []
+    for key, val in genre_dict.items():
+        genre_count_list.append(val)
+    genre_count_list.sort()
+    for key, val in genre_dict.items():
+        if val == genre_count_list[-1]:
+            return key 
+
+def get_unique_watched(user_data):
+    user_watched_set = set()
+    for movie in user_data['watched']:
+        user_watched_set.add(movie['title'])
+    
+    friend_watched_set = set()
+    for movie in user_data['friends']:
+        for val in movie['watched']:
+            friend_watched_set.add(val['title'])
+            
+    user_watched_not_friend_watched = user_watched_set.difference(friend_watched_set)
+    
+    movie_dict_list = []
+    movie_dict = {}
+    for i in user_watched_not_friend_watched:
+        movie_dict_list.append({'title':i})
+    
+    return movie_dict_list
+
+def get_friends_unique_watched(user_data):
+    user_watched_set = set()
+    for movie in user_data['watched']:
+        user_watched_set.add(movie['title'])
+    
+    friend_watched_set = set()
+    for movie in user_data['friends']:
+        for val in movie['watched']:
+            friend_watched_set.add(val['title'])
+    
+    friend_watched_not_user_watched = friend_watched_set.difference(user_watched_set)
+    
+    movie_dict_list = []
+    movie_dict= {}
+    for i in friend_watched_not_user_watched:
+        movie_dict_list.append({'title':i})
+    return movie_dict_list
+
+def get_available_recs(user_data):
+  
+    user_hosts = user_data['subscriptions']
+    user_watched = user_data['watched']
+    friends_watched = user_data['friends']
+    recommendations = []
+    
+    for movie_dict in friends_watched:
+        #print(movie_dict)
+        for movie in movie_dict['watched']:
+            print(movie,"HERE")
+            #for key,val in movie.items():
+                #print(key,val)
+                #print(movie_dict['title'])
+            if movie['host'] in user_data['subscriptions']:
+                if movie['title'] not in user_watched:
+                        if movie not in recommendations:
+                            recommendations.append(movie)
+    return recommendations
+
+
+def get_new_rec_by_genre(user_data):
+       
+    recommendations = []
+    user_watched = user_data['watched']
+    
+    if len(user_watched) == 0:
+        return []
+    user_most_watched_genre = get_most_watched_genre(user_data)
+    #print(user_last_watched_genre,'most watched genre')
+    friends = user_data['friends']
+    for friend in friends:
+        #print(friend)
+        for movie in friend['watched']:
+            #print(movie['genre'],'movie')
+            
+            if movie['genre'] == user_most_watched_genre:
+                if movie not in recommendations:
```

Comment:
Consider using a compound conditional 
```suggestion
            if movie['genre'] == user_most_watched_genre and movie not in recommendations:
```